lib/backend/dynamo/dynamodbbk.go

```diff
@@ -654,10 +657,17 @@ func (b *Backend) getTableStatus(ctx context.Context, tableName string) (tableSt
 // documentation in case users want to set up DynamoDB tables manually. Edit the
 // following docs partial:
 // docs/pages/includes/dynamodb-iam-policy.mdx
-func (b *Backend) createTable(ctx context.Context, tableName string, rangeKey string) error {
-	pThroughput := dynamodb.ProvisionedThroughput{
-		ReadCapacityUnits:  aws.Int64(b.ReadCapacityUnits),
-		WriteCapacityUnits: aws.Int64(b.WriteCapacityUnits),
+func (b *Backend) createTable(ctx context.Context, tableName string, rangeKey string, onDemand bool) error {
+	var pThroughput *dynamodb.ProvisionedThroughput
+	var billingMode *string
+	if onDemand {
+		billingMode = aws.String(dynamodb.BillingModePayPerRequest)
+	} else {
+		pThroughput = &dynamodb.ProvisionedThroughput{
+			ReadCapacityUnits:  aws.Int64(b.ReadCapacityUnits),
+			WriteCapacityUnits: aws.Int64(b.WriteCapacityUnits),
+		}
+		billingMode = aws.String(dynamodb.BillingModeProvisioned)
```

Comment:
Is this a breaking change? We don't set the billing mode to provisioned, right now.